onboarding/src/main/res/values/strings.xml

```diff
@@ -17,7 +17,18 @@
     <string name="manual_server_clear_url">Clear server url</string>
     <string name="manual_server_wrong_url">Invalid Home\u00A0Assistant address</string>
     <string name="connection_screen_malformed_url">Verify that your Home\u00A0Assistant address is correct</string>
-    <string name="local_first_title">Local by default.\nRemote when you’re ready.</string>
+    <string name="local_first_title">Local by default.\nRemote when you\'re ready.</string>
     <string name="local_first_content">Your Home\u00A0Assistant is only accessible on your home network. To control your smart home from anywhere, you can set up remote access later in your settings.\n\nFor now, you\'re securely connected to your local network.</string>
     <string name="local_first_next">Got it</string>
+    <string name="location_sharing_title">Use this device\'s location for automations</string>
+    <string name="location_sharing_content">Location sharing enables powerful automations, such as turning off the heating when you leave home. This option shares the device\'s location only with your Home Assistant system.\n\nThis data stays in your home and is never sent to third parties. It also helps strengthen the security of your connection to Home Assistant.</string>
```

Comment:
> It also helps strengthen the security of your connection to Home Assistant.

This is added at the end without details, raising questions. How does it help strengthen the security? I understand trying to keep it simple, but maybe you need to add some text from the second screen here as well.

Reply:
I think we can keep the copy like this for now. I know @laupalombi spent a lot of time already on this screen.